e2e/runner/zeta.go

```diff
@@ -95,11 +95,11 @@ func (r *E2ERunner) DepositZeta() ethcommon.Hash {
 	amount := big.NewInt(1e18)
 	amount = amount.Mul(amount, big.NewInt(100)) // 100 Zeta
 
-	return r.DepositZetaWithAmount(r.EVMAddress(), amount)
+	return r.DepositZetaWithAmount(r.EVMAddress().Bytes(), amount)
 }
 
 // DepositZetaWithAmount deposits ZETA on ZetaChain from the ZETA smart contract on EVM with the specified amount
-func (r *E2ERunner) DepositZetaWithAmount(to ethcommon.Address, amount *big.Int) ethcommon.Hash {
+func (r *E2ERunner) DepositZetaWithAmount(to []byte, amount *big.Int) ethcommon.Hash {
```

Comment:
Let’s keep the E2E with `ethcommon.Address`, rather than refactoring to `[]byte` solely to test an invalid EVM address. I suggest we handle this case change separately.

Reply:
I think using []byte or [20]byte here is a good idea; it keeps the function more generic